TTS/tts/models/base_tts.py

```diff
@@ -199,6 +209,13 @@ def get_data_loader(
             if hasattr(self, "make_symbols"):
                 custom_symbols = self.make_symbols(self.config)
 
+            if hasattr(self, "language_manager"):
```

Comment:
I think speaker manager can handle the language mapping too

Reply:
Also, BaseTTS should not include language settings if we don't plan to support multiple language training in all the TTS models. 

It is better to create BaseMultiLangTTS inheriting BaseTTS